registry/api/v2/errors.go

```diff
@@ -40,16 +40,6 @@ var (
 		HTTPStatusCode: http.StatusBadRequest,
 	})
 
-	// ErrorCodeSizeInvalid is returned when uploading a blob if the provided
-	ErrorCodeSizeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
I removed it because I noticed it was unused. Probably should have made a separate PR. Does it still require a spec change if it was never used?